pkg/eigenState/stakerShares/stakerShares.go

```diff
@@ -151,14 +182,16 @@ func (ss *StakerSharesModel) handleStakerDepositEvent(log *storage.TransactionLo
 		return nil, xerrors.Errorf("Failed to convert shares to big.Int: %s", outputData.Shares)
 	}
 
-	return &StakerShareDeltas{
-		Staker:          stakerAddress,
-		Strategy:        outputData.Strategy,
-		Shares:          shares.String(),
-		StrategyIndex:   uint64(0),
-		LogIndex:        log.LogIndex,
-		TransactionHash: log.TransactionHash,
-		BlockNumber:     log.BlockNumber,
+	return &StateDiff{
```

Comment:
Why? StateDiff only contains a StakerShareDeltas struct so why is it needed at all?

Reply:
https://github.com/Layr-Labs/sidecar/pull/68/commits/849bd0db77bd553611d6ccdf776de875882dc9c5